CMakeLists.txt

```diff
@@ -111,7 +111,7 @@ set(NABO_SRC
 	nabo/kdtree_cpu.cpp
 	nabo/kdtree_opencl.cpp
 )
-set(SHARED_LIBS "false" CACHE BOOL "To build shared (true) or static (false) library")
+set(SHARED_LIBS FALSE CACHE BOOL "To build shared (true) or static (false) library")
```

Comment:
s/"To build shared (true) or static (false) library"/"Set to TRUE to build shared library"